vite-project/src/components/Card.tsx

```diff
@@ -37,24 +41,87 @@ export default function Cards({ data }: Props) {
-            <h3>Date for note:</h3> <p>{data?.date}</p>
-          </ul>
-        </div>
-        <div onClick={handleToggleFavorite} style={{ cursor: "pointer" }}>
-          {isFavorite ? (
-            <FontAwesomeIcon icon={faHeart} className="heart-filled" />
-          ) : (
-            <FontAwesomeIcon icon={faHeart} className="heart-empty" />
-          )}
-        </div>
-      </div>
-    </div>
+    <ThemeProvider theme={Mytheme}>
+      <Box
+        className="area"
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          marginBottom: "1rem",
+          justifyContent: "center",
+          width: "100%",
+          padding: "1rem 0;",
+          marginLeft: "10rem",
+          paddingTop: "4rem",
+        }}
+      >
+        <Box
+          className="cards"
+          sx={{
+            position: "relative",
+            width: "24rem",
+            height: "24rem",
+            borderRadius: "3rem",
+            backgroundColor: "background.default",
+            boxShadow: "inset 0 0 10px rgba(0, 0, 0, 0.5)",
+            [theme.breakpoints.down("lg")]: {
+              width: "30.25rem",
+              height: "30.3rem",
+              marginRight: "56rem",
+              alignItems: "center",
+            },
+          }}
+        >
+          <Box
+            className="thefront"
+            sx={{
+              position: "absolute",
+              width: "100%",
+              height: "100%",
+              display: "flex",
+              textAlign: "center",
+              alignItems: "center",
+              marginRight: "3rem",
+            }}
+          >
+            <List
+              sx={{
+                flex: "1 1 48%",
+                listStyleType: "none",
+              }}
+            >
+              <Typography variant="h4">Name of the song:</Typography>
+              <Typography variant="h6">{data?.name}</Typography>
+              <Typography variant="h4">Genre:</Typography>
+              <Typography variant="h6">{data?.genre}</Typography>
+              <Typography variant="h4">Artist:</Typography>
+              <Typography variant="h6">{data?.artist}</Typography>
+              <Typography variant="h4">Date for note:</Typography>
+              <Typography variant="h6">{data?.date}</Typography>
+            </List>
+          </Box>
+
+          <FavoriteIcon
+            onClick={handleToggleFavorite}
+            className="heart-filled"
+            sx={{
+              width: "4rem",
+              height: "4rem",
+              marginTop: "-1.5rem",
+              cursor: "pointer",
+              marginLeft: "10rem",
+              borderRadius: "0.5rem",
+              color: isFavorite ? "action.hover" : "action.active",
+              [theme.breakpoints.down("lg")]: {
+                width: "5.25rem",
+                height: "5.3rem",
+                marginLeft: "12rem",
```

Comment:
Use MUI sizings instead of rem -> https://mui.com/system/sizing/
Do that for every place rem or pixels are used